curator/README.md

```diff
@@ -46,12 +61,19 @@ For example, using::
       runhour: 0
       runminute: 0
       timezone: America/New_York
+
+    ^project\..+\-dev.*$:
```

Comment:
Are all regular expressions also valid YAML keys?  If not, then we will need to change the structure (and maybe we should change it anyway, just to be safe):
```yaml
myapp-dev:
  delete:
...
.regex:
  - pattern: "^project\..+\-dev.*$"
    delete:
      days: 1
```
In general: the `.regex` element is a List - each list element has a `pattern` key - the value of this key is a YAML string properly escaped e.g. you can't have a string value like `{..something`.  See http://www.yaml.org/spec/1.2/spec.html#id2786942 for a definition of a yaml string scalar - or a block scalar e.g.
```yaml
.regex:
  - pattern: '^project\..+\-dev'
    delete:
      days: 3
```
or I suppose a block scalar would work too
```yaml
.regex:
  - pattern: |
      ^project\..+\-dev
    delete:
      days: 3
```